session.py

```diff
@@ -29,13 +29,9 @@
 class GaeSession:
 	"""Store Sessions inside the Big Table/Memcache"""
 	kindName = "viur-session"
-	sameSite = "lax"
-
-	def __init__(self):
-		super().__init__()
-
-		self.plainCookieName = f"viurHttpCookie_{utils.projectID}"
-		self.sslCookieName = f"viurSSLCookie_{utils.projectID}"
+	sameSite = "lax"  # Either None (dont issue sameSite header), "none", "lax" or "strict"
+	sessionCookie = True  # If True, issue the cookie without a lifeTime (will disappear on browser close)
+	cookieName = f"viurCookie_{utils.projectID}"
```

Comment:
For above 3 lines, is there a way or possibility to customize or override the default values with minor effort?

Reply:
That's why there left as class variables instead of being hard-coded into the one line that uses them. Shall we move them to conf? (We might want to decide if we put everything in conf - or if me move stuff that's rarely and only once used to the class it belongs to)

Reply:
hmm okay then it should satisfy this. I'll approve for now.